### DISCUSSION TITLE
How to get numbers of offers and sellers name foror a specific ASIN ?

### DISCUSSION THREAD

**Discussion by zenplayx** [amzn/selling-partner-api-models#3516] [General]
Hello.

For specific ASIN, I would like to retrieve the number of offers and sellers name ?

How can I do that with the API ?

Thank you.

**Comment by 0xs3n4** [comment]
You did it? I have the same need.
I'm new with the sp-api,i read all of documentation, it is very hard, very confuses...

**Comment by 0xs3n4** [comment]
Hello my friend, look at me here again rsrsrsrs, after many searching in doc and git hub discussions, i find this API 
[API-PRINCING-V0](https://developer-docs.amazon.com/sp-api/docs/product-pricing-api-v0-reference)

I assume I didn't consider this option, because we have a more updated PRODUCt-PRINCING, so I didn't look at the old one because I thought it would be discontinued... The lesson is to look at all the versions hehehe
